security/acme-client/src/opnsense/mvc/app/views/OPNsense/AcmeClient/certificates.volt

```diff
@@ -354,13 +339,13 @@ POSSIBILITY OF SUCH DAMAGE.
             {
                 if (gridParams['revoke'] != undefined) {
                     var uuid=$(this).data("row-id");
-                    stdDialogConfirmation('Revoke selected certificate?',function() {
+                    stdDialogConfirm('{{ lang._('Confirmation Required') }}','{{ lang._('Revoke selected certificate?') }}','{{ lang._('Yes') }}', '{{ lang._('Cancel') }}',function() {
```

Comment:
1) space after commas
2) this like is in my opinion too long - maybe add line breaks after the commas and align the parameters to a block.